src/Initializer/Parameters/ModelParameters.h

```diff
@@ -41,6 +42,14 @@ constexpr bool isModelAnisotropic() {
 #endif
 }
 
+constexpr bool isModelNonLinear(){
```

Comment:
The equation is called `damaged-elastic`. So this should be reflected here.

Reply:
Modified